docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -305,6 +305,15 @@ where `OOM` stands for Out Of Memory.
 
 ## Opaque integer resources (Alpha feature)
 
+{% include feature-state-deprecated.md %}
+
+**Note:** Opaque Integer Resources will be removed in version 1.9.
```

Comment:
Move this note below https://github.com/kubernetes/kubernetes.github.io/pull/5399/files?diff=unified#diff-4c9401ed6b037e1adb958cbce20630c7L321.